### PR TITLE
Add kanban article.

### PR DESCRIPTION
**Pull request by willfish**

### CODE REVIEW

File: posts/kanban_task_switching.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Calculating the cost of task switching: why we use Kanban here at MyDrive.
+author: william
+date: "2015-03-09"
+description: Some thoughts about the detrimental affects of task switching and why Kanban is so awesome for eliminating it and maximising flow.
```
Comment (jfharden) [inline posts/kanban_task_switching.md]: effects not affects

File: posts/kanban_task_switching.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+A beginning is a very important time in a person's career. This has recently been highlighted to me in terms
+of the bad habits I developed in my previous job. Coming from a Linux System Administrator background, where an ad
+hoc attitude was taken to the work I was given, I struggled to adapt to the work-on-one-thing-at-a-time mentalility 
+here [@MyDrive](https://twitter.com/_mydrive). I have found myself switching between different tasks and noticed that
+this had quite a significantly adverse affect on my ability to deliver quality output when compared to other members of
+the team. And it was not making me feel good about my contributions.
+
+Luckily for me, the team here is incredibly proactive and are always willing to educate and improve on
+how we work as a unit. We have already had detailed sessions on Kanban (today's session inspired this
```
Comment (mhefferan) [inline posts/kanban_task_switching.md]: You could mention the coin game? Here is a stricter version, but you could talk about the simplified version we did? https://darynholmes.wordpress.com/2011/08/02/the-x-penny-game/

**Comment by mhefferan** [comment]
I think in some bits the tone sounds like you are being a bit hard on yourself.

File: posts/kanban_task_switching.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+date: "2015-03-09"
+description: Some thoughts about the detrimental affects of task switching and why Kanban is so awesome for eliminating it and maximising flow.
+tags: ["kanban", "flow", "task switching", "team work"]
+categories: ["development", "programming", "continuous improvement", "kanban"]
+---
+
+The aim of this post is to suggest that task switching is cognitively expensive and that Kanban as an agile methodology
+lends itself to reducing the cost of task switching.
+
+A beginning is a very important time in a person's career. This has recently been highlighted to me in terms
```
Comment (mhefferan) [inline posts/kanban_task_switching.md]: Refactor the first few sentences, what you are trying to say is good but it sounds a bit rambling. Less is more.

**Comment by gavinheavyside** [comment]
You'll want to add yourself to the authors config here: https://github.com/mydrive/mydrive.github.io/blob/gh-pages/config.yml#L17-L25

**Comment by gavinheavyside** [comment]
:shipit:

**Pull request merged**